config/secrets.yml

```diff
@@ -119,6 +119,8 @@ development:
   sentry:
       dsn: <%= ENV['SENTRY_DSN'] || 'invalid' %>
 
+  posthog_project_api_key: <%= ENV['POSTHOG_PROJECT_API_KEY'] || 'phc_n9UdrQ7MA5hF21CqmbGWoSxM1LfzxpcCxcjUBGfcdrE' %>
```

Comment:
Well, just make sure this key is not valid for anything, as this repo is public.

Reply:
this is the test key (and is available in the javascript, anyway); there is a separate production key in the parameter store
they key is write only, so it being a test key is okay in this instance (subject to change if someone starts hammering our posthog instance of course) - this just makes it easier to change